frontend/src/app/statistics/page.tsx

```diff
@@ -71,10 +49,18 @@ export default function Statistics() {
   );
 
   const handleSubmit = () => {
-    if (!state || !district || !parameter || !startingYear || !endingYear || !infoType) {
+    if (
+      !state ||
+      !district ||
+      !parameter ||
+      !startingYear ||
+      !endingYear ||
+      !infoType
+    ) {
       alert("Please fill all the fields.");
       return;
     }
+    setGifState(true);
     console.log(state, district, parameter, startingYear, endingYear, infoType);
     axios
       .get("http://localhost:8080/api/statistics", {
```

Comment:
@Kushdapush We will create separate environment files for production and development, define appropriate variables in each, and use these variables within our code accordingly. 
You may refer to this : [Reference link for setting up environment variables](https://github.com/vercel/next.js/discussions/12077#discussioncomment-92496)

Reply:
Alright, ill make respective env files for local, staging and production